ci/script.sh

```diff
@@ -1,80 +0,0 @@
-set -ex
-
-. $(dirname $0)/env.sh
-
-gist_it() {
-    gist -d "'$TARGET/rustc-builtins.rlib' from commit '$TRAVIS_COMMIT' on branch '$TRAVIS_BRANCH'"
-    echo "Disassembly available at the above URL."
-}
-
-build() {
-    if [[ $WEAK ]]; then
-        $CARGO build --features weak --target $TARGET
-        $CARGO build --features weak --target $TARGET --release
-    else
-        $CARGO build --target $TARGET
-        $CARGO build --target $TARGET --release
-    fi
-}
-
-inspect() {
-    $PREFIX$NM -g --defined-only target/**/debug/*.rlib
-
-    set +e
-    $PREFIX$OBJDUMP -Cd target/**/release/*.rlib | gist_it
-    set -e
-
-    # Check presence/absence of weak symbols
-    if [[ $WEAK ]]; then
-        local symbols=( memcmp memcpy memmove memset )
-        for symbol in "${symbols[@]}"; do
-            $PREFIX$NM target/$TARGET/debug/deps/librlibc-*.rlib | grep -q "W $symbol"
```

Comment:
I haven't re-added this test, but I figure that it's something that should be tested in rlibc rather than here.